woocommerce/woocommerce
# Check for the WooCommerce Admin class before adding notice

### All Submissions:






### Changes proposed in this Pull Request:

Check to see if WooCommerce Admin FeaturePlugin class exists before adding notice instead of checking for default plugin folder.

This is a minor issue, but can happen if the folder is not using the default name.

<img width="1164" alt="Screen Shot 2020-01-13 at 5 38 59 PM" src="https://user-images.githubusercontent.com/10561050/72246569-f6302500-362d-11ea-978e-d797c2639464.png">

Closes #25394

### How to test the changes in this Pull Request:

1. Install wc-admin.
1. Rename to anything other than woocommerce-admin.
1. Trigger `woocommerce_installed` or switch themes to trigger the notice.
1. Make sure the above notice isn't shown.

### Other information:




### Changelog entry

> Check for WooCommerce Admin feature plugin class before adding install notice
